### DISCUSSION TITLE
hunting transmitter error

### DISCUSSION THREAD

**Discussion by dbfreud** [NightscoutFoundation/xDrip#3086] [Bug reports]
I am setting up xdrip for first time. I thought I had connected, and xdrip showed a few hours of bg history but can't get past "hunting transmitter" message. 
Devices are
![Screenshot_20230917-093309_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/145223961/4bd7356d-a444-4571-8de3-f2c0718916b4)
![Screenshot_20230917-091958_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/145223961/078e1429-b146-49b1-a954-32ee202d967d)
 as follows: Dexcom G7, Samsung Galaxy S10 (Android version 12)

**Comment by Navid200** [comment]
Are all settings correct?
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

**Comment by dbfreud** [comment]
Thanks for your reply.  I just went through these again (I set up
yesterday) and confirmed.

The only one I can't find in the re-check is disable battery optimization.
I recall setting that up yesterday but I can't find it again. I followed
the instructions to go to apps then xdrip+ but there's no "advanced" option
to select from  within that menu.  there is the option to allow xdrip to
change system settings, which is currently not permitted but i can select

**Comment by dbfreud** [comment]
I just found it and confirmed that battery usage is unrestricted.  So all
settings seem correct

**Reply by Navid200** [reply]
Does Dexcom app work?

**Comment by dbfreud** [comment]
yes it has worked fine  I uninstalled before I started xdrip per
instruction

**Comment by dbfreud** [comment]
per pic, it looks like it connected to capture a few hours BG history but I
don't think it ever read in real time

**Reply by Navid200** [reply]
What happens if you swipe right, to go to the classic status page, and tap on forget device?
Then, go back to the G5/G6/G7 status page and wait for a pair request.  After you approve, will it work then?

**Comment by dbfreud** [comment]
forget devices is in grey and does not change when I select it. I'll see if
anything happens

**Reply by Navid200** [reply]
In that case, go to Android settings and look under previously connected or connected devices.  Forget all Dexcom-related devices.

**Comment by dbfreud** [comment]
just sent pics of both status screens

**Comment by dbfreud** [comment]
yup just did that, makes sense. is there a way to prompt xdrip to pair?

**Reply by Navid200** [reply]
After you forget devices, it will show a pair request in up to 5 minutes.

**Reply by Navid200** [reply]
Unless it is already paired with Dexcom app on another device?

**Comment by dbfreud** [comment]
what if I just gave up on this sensor and tried with a new one?

**Comment by dbfreud** [comment]
oh I did earlier in the day before writing selected stop sensor but it's
still showing the code for the sensor I originally entered

**Reply by Navid200** [reply]
You enter the 4-digit code from the inserter as the transmitter ID.  The transmitter ID is under settings.

**Comment by dbfreud** [comment]
yup I did that yesterday. later today when I have access to a new sensor I
can try again and see if that works

**Reply by Navid200** [reply]
I'm not sure what you mean.  But, it's up to you.
I wanted to figure out why your system right now is not working.  You just mentioned that the user id is of the previous sensor.  
Why?  Fix it.

**Comment by dbfreud** [comment]
I thought the next step was to try with a new sensor but don't presently
have access to another one.

**Comment by dbfreud** [comment]
I have limited cell service right now which is why I'm delayed in
responding to you. copying in Ken who has better service. if there's
something else you're advising doing right now other than starting with a
new sensor (which I can only do later) please let me know. thanks

**Reply by Navid200** [reply]
You have posted a single G5/G6 G7 status page.  it shows an inserter code of 6470.  Is that the code from the sensor you are using right now?

**Comment by dbfreud** [comment]
yes but as I said I tried to stop that sensor earlier today. nothing
changed when I did that

**Reply by Navid200** [reply]
What do you mean stop.  You don't stop a G7.  Why would you?

**Comment by dbfreud** [comment]
there is an option in xdrip to stop sensor. I believe I selected that this
morning when I was trying to get it to work but drip is still showing 6470
as connected. you directed me in this string to forget prior dexcom
Bluetooth connections which I did

**Reply by Navid200** [reply]
The problem with what is on the status page is that it is showing hunting for transmitter, which means there is no connectivity.
I wanted you to forget device to see if you get a pair request.
But, you said the transmitter id was wrong.
Then, you said you are uploading more pages.  But, I saw nothing new.
I still don't know if 6470 is the device you have on or the previous device.

I am happy to leave this al.one because it is very hard to communicate.

**Comment by dbfreud** [comment]
my apologies if my replies are unclear. I would be grateful for your
continued help to get this working and will follow any directions you
offer. thank you

**Reply by Navid200** [reply]
Is 6470 from the current device you have on now?
Or, is 6470 the one you used previously?

**Comment by dbfreud** [comment]
that's the only one I have entered. it's still on and has got five days left

**Reply by Navid200** [reply]
So, Dexcom app works fine.  But, when you uninstall it, xDrip shows hunting transmitter?

**Reply by Navid200** [reply]
With the Dexcom app uninstalled, please enable detail logging as explained here: https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

Wait 15 minutes.
Then, upload logs as explained and enter an identifying note.
Then, post here with the identifying note so that the person who receives the logs can find this thread to see the description.

Then, you can switch back to Dexcom app and wait for feedback.

**Comment by dbfreud** [comment]
yes

**Comment by dbfreud** [comment]
will do thanks

**Comment by dbfreud** [comment]
sorry couldn't follow these instructions.  Is there another way to try
again to try to reset to give it another chance to get the transmitter
connected?  I understand I can't start a new sensor until the transmitter
has connected

**Reply by Navid200** [reply]
If Dexcom app works fine, so should xDrip as long as the settings are correct..
You said all settings were correct and xDrip was not working.
Then, we need a report so that we can fix it.

I explained how you can submit a report and wait for feedback.
The feedback either will explain to you what you have done wrong, or offer a new xDrip release to try.  But, none of that is going to happen if you don't submit logs as explained.

**Comment by dbfreud** [comment]
ok I have gone through the instructions again.  Please advise where I will
find auto configure

**Comment by dbfreud** [comment]
the instructions direct me to use xdrip auto configure to scan the code,
but I can't find that

**Reply by Navid200** [reply]
Use the main menu (hamburger menu) at the top left of the main screen and tap on Settings.
You will see Auto Configure with a QR code icon beside it.

**Comment by dbfreud** [comment]
submitted

**Reply by Navid200** [reply]
Thanks

**Comment by dbfreud** [comment]
Letting you know that Xdrip began working. When I reinstalled Dexcom as you
directed I didn't remove Xdrip. Seven hours ago, Xdrip picked up the
transmitter and Dexcom stopped doing so.
Tthe only change I am aware of is I had removed the Xdrip watch face at
about the time the app started working again. As previously reported,
that's what happened when I initially installed the app; it was working
until I installed the Xdrip watch face on my Galaxy Classic 4. I have now
uninstalled the Dexcom app and reinstalled the Xdrip watch face. So far
both are functioning.

I'll write if it stops again.
